pxr/usd/usdGeom/plugInfo.json

```diff
@@ -390,6 +390,14 @@
                         ], 
                         "schemaKind": "abstractTyped"
                     }
+                },
+                "Validators": {
+                    "StageMetadataChecker": {
+                        "doc": "All stages should declare their 'upAxis' and 'metersPerUnit' and stages meant for consumer-level packaging should always have upAxis set to 'Y'."
```

Comment:
The secondary bits of "consumer-level packaging should always have upAxis set to 'Y', wasnt that a arkit rule requirement from ComplianceChecker. I would remove this statement. (https://github.com/PixarAnimationStudios/OpenUSD/blob/dev/pxr/usd/usdUtils/complianceChecker.py#L213-L217)

It looks like you code is also not doing this check, which is correct!

Reply:
good catch, updated